raiden-dapp/src/locales/en.json

```diff
@@ -304,6 +304,45 @@
       "label": "Amount"
     }
   },
+  "transfer-action": {
+    "confirm-button-label": "Confirm Payment",
+    "steps": {
+      "transfer": {
+        "title": "Transfering tokens",
+        "description": "Please wait for your payment"
+      }
+    }
+  },
+  "channel-deposit-and-transfer-action": {
+    "confirm-button-label": "Deposit & Pay",
+    "steps": {
+      "deposit": {
+        "title": "Depositing tokens to channel",
+        "description": "Please do not close the browser and confirm the transaction with your wallet."
+      },
+      "transfer": {
+        "title": "Transfering tokens",
+        "description": "Please wait until tokens have ben successfully sent."
+      }
+    }
+  },
+  "channel-open-and-transfer-action": {
+    "confirm-button-label": "Open Channel & Pay",
```

Comment:
"confirm-button-label": "Open Channel and Pay"

Reply:
✔️